Gemfile

```diff
@@ -1,54 +1,22 @@
 source 'https://rubygems.org'
 
 ruby '1.9.3'
-gem 'rails', '3.2.16'
-
-# Bundle edge Rails instead:
-# gem 'rails', :git => 'git://github.com/rails/rails.git'
 
+gem 'rails', '3.2.16'
 gem 'sqlite3'
 
-# Gems used only for assets and not required
-# in production environments by default.
 group :assets do
-  gem 'sass-rails',   '~> 3.2.3'
   gem 'coffee-rails', '~> 3.2.1'
-
-  # See https://github.com/sstephenson/execjs#readme for more supported runtimes
-  # gem 'therubyracer', :platforms => :ruby
-
+  gem 'angularjs-rails'
+  gem 'angular-ui-bootstrap-rails'
   gem 'uglifier', '>= 1.0.3'
 end
 
 group :development, :test do
   gem 'debugger'
-  gem 'jasmine-rails' # JavaScript/CoffeeScript -- is this necessary?
-end
-
-group :test do
-  gem 'rspec-rails', '2.14'
   gem 'simplecov', :require => false
-  gem 'cucumber-rails', :require => false
-  gem 'cucumber-rails-training-wheels'
   gem 'database_cleaner'
-  gem 'autotest-rails'
-  gem 'metric_fu'
+  gem 'jasmine-rails'
+  gem 'rspec-rails', '2.14'
+  gem 'cucumber-rails', :require => false
 end
-
-gem 'jquery-rails'
-gem 'haml'
```

Comment:
Maybe add `gem 'less-rails'` and `gem 'therubyracer'` too? (I think they're needed for compiling less)